client/src/includes/initIconSprite.ts

```diff
@@ -0,0 +1,45 @@
+export const initIconSprite = (
```

Comment:
```suggestion
/**
 * Loads sprite data from either Local Storage or via async fetch.
 * Ensures the sprite data is backed up when pulled in from API.
 */
export const initIconSprite = (
```

Nice to have a JSDoc describing the function, even a simple description helps 'space out' the code for readability and helps for understanding what is happening.